Makefile

```diff
@@ -450,11 +450,11 @@ devenv-mysql:
 .PHONY: protobuf
 protobuf: ## Compile protobuf definitions
 	bash scripts/protobuf-check.sh
-	go install google.golang.org/protobuf/cmd/protoc-gen-go
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.36.5
```

Comment:
I pinned the version, as the result of using an unpinned version can be drift, as the format changes frequently.